x-pack/legacy/plugins/canvas/canvas_plugin_src/renderers/time_filter/components/time_picker_popover/time_picker_popover.tsx

```diff
@@ -22,7 +23,15 @@ export interface Props {
 
 export const TimePickerPopover: FunctionComponent<Props> = ({ from, to, onSelect }) => {
   const button = (handleClick: (event: MouseEvent<HTMLButtonElement>) => void) => (
-    <button className="canvasTimePickerPopover__button" onClick={handleClick}>
+    <button
+      className="canvasTimePickerPopover__button"
+      aria-label={
+        'Displaying data ' +
+        formatDuration(from, to) +
+        '. Click to open a calendar tool to select a new time range.'
```

Comment:
def a nit but any reason why you're not using [template literals](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Template_literals) for your strings that need data? Same effect is achieved but I think template literals are a little cleaner

Reply:
I figured there was a new, fancy way to do this and that someone would tell me. I'll switch it to template literals.